### PR TITLE
added button to list of students

### PR DESCRIPTION
**Pull request by carolarruda**

### CODE REVIEW

**Comment by carolarruda** [comment]
[TeacherView - list of Students](https://trello.com/c/ayy4Z2Vv/76-teacherview-list-of-students)

File: src/pages/dashboard/style.css [hunk 1]
```diff
@@ -69,3 +69,7 @@ aside {
 
   }
```
[the remark this answers is not in the document]
Comment (dearshrewdwit) [reply src/pages/dashboard/style.css:71]: @ample-samples try to phrase this in a different way - better yet ask for clarification first :)

"what's the reasoning for this?"

[the remark this answers is not in the document]
Comment (ample-samples) [reply src/pages/dashboard/style.css:71]: @dearshrewdwit Hi Ed, this is just banter as were competing for most commits on the project :P

[the remark this answers is not in the document]
Comment (dearshrewdwit) [reply src/pages/dashboard/style.css:71]: Well then! Comment away, @ample-samples

**Review by Gabriel-Rosengren — APPROVED** [review]

File: src/pages/dashboard/style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (ample-samples) [reply src/pages/dashboard/style.css:71]: Carolina your PR looks fine to me

[the remark this answers is not in the document]
Comment (carolarruda) [reply src/pages/dashboard/style.css:71]: @ample-samples, really? At least approve my PR xD

**Review by ample-samples — APPROVED** [review]
It's ok

**Pull request merged**